### PR TITLE
www-apps/kibana-bin: bump to 5.5.1, drop old

### PR DESCRIPTION
**Pull request by hydrapolic**
https://bugs.gentoo.org/show_bug.cgi?id=617718

Package-Manager: Portage-2.3.6, Repoman-2.3.2

### CODE REVIEW

File: www-apps/kibana-bin/metadata.xml [hunk 1]
```diff
@@ -5,10 +5,6 @@
     <email>hydrapolic@gmail.com</email>
     <name>Tomáš Mózes</name>
   </maintainer>
-  <maintainer type="person">
```
Comment (monsieurp) [inline www-apps/kibana-bin/metadata.xml]: @hydrapolic hm. can we get said maintainer to sign off on this removal?

**Comment by hydrapolic** [comment]
Just by checking the git log there seems no activity and the last post in the bugs regarding kibana was almost 2 years ago. I'll try to mail him and wait for an answer.

**Comment by hydrapolic** [comment]
@monsieurp, I'm also fine by merging the bump/drop and waiting for a reply from Ferenc.

**Comment by monsieurp** [comment]
@hydrapolic let's do that. please file a bug to get him removed and assign it to proxy-maint as well as him. let's set a 30 day timeout after which we'll remove him from the metadata.xml file if he doesn't reply, ok?

**Comment by ferki** [comment]
@monsieurp: as asked previously by @hydrapolic via email, I'm fine with dropping me from list of maintainers. I'll ask for readding myself whenever I'll become more active again around the ELK stack.

Thanks for your work on it! :+1:

**Comment by hydrapolic** [comment]
Thanks

**Pull request closed without merging**